Library/Formula/mono.rb

```diff
@@ -30,7 +34,13 @@ def install
       --prefix=#{prefix}
       --enable-nls=no
     ]
-    args << "--build=" + (MacOS.prefer_64_bit? ? "x86_64": "i686") + "-apple-darwin"
+
+    if build.with? "64bit"
+      args << "--build=x86_64-apple-darwin"
+    else
+      # no option specified
+      args << "--build=" + (MacOS.prefer_64_bit? ? "x86_64": "i686") + "-apple-darwin"
```

Comment:
Sorry, I'm an idiot who can't read. I apologise. My question though: what platform are you on that means you need to create the 64-bit option to force it to be enabled?